src/main/default/customMetadata/Trigger_Handler.testHandle.md-meta.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomMetadata xmlns="http://soap.sforce.com/2006/04/metadata" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <label>testHandle</label>
+    <protected>false</protected>
+    <values>
+        <field>After_Delete__c</field>
+        <value xsi:type="xsd:boolean">false</value>
+    </values>
+    <values>
+        <field>After_Insert__c</field>
+        <value xsi:type="xsd:boolean">false</value>
+    </values>
+    <values>
+        <field>After_Undelete__c</field>
+        <value xsi:type="xsd:boolean">false</value>
+    </values>
+    <values>
+        <field>After_Update__c</field>
+        <value xsi:type="xsd:boolean">false</value>
+    </values>
+    <values>
+        <field>Auto_Bind__c</field>
```

Comment:
this is fluff, just don't create handler records, or disable them if you need to ... only use case this supports is wanting to turn off all auto-bound handlers but keep all directly bound, i can't see that coming up frequently enough to justify an additional config point